pkgs/by-name/ro/rose-pine-cursor/package.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
```

stdenv pulls in a C compiler which isn't needed to build this package.

```suggestion
, stdenvNoCC
```